steward/src/cellars/adaptors/aave_v3.rs

```diff
@@ -171,9 +153,42 @@ pub(crate) fn aave_v3_debt_token_adaptor_v1_calls(
     Ok(calls)
 }
 
+pub(crate) fn aave_v3_debt_token_adaptor_v1_flash_loan_calls(
+    params: AaveV3DebtTokenAdaptorV1FlashLoanCalls,
+) -> Result<Vec<Bytes>, Error> {
+    let mut calls = Vec::new();
+    for c in params.calls {
+        let function = c
+            .function
+            .ok_or_else(|| sp_call_error("function cannot be empty".to_string()))?;
+
+        let aave_v3_debt_token_adaptor_v1_flash_loan::Function::FlashLoan(p) = function;
```

Comment:
Is this backwards or some kind of weird rust syntax I don't remember? Where does `p` come from, or is this defining it?

Reply:
it's defining `p`, newish syntax to avoid a match block and keep the var in cope. normally it could be followed by an `else` block but there is only one variant. this will change when i address your other comment about not needing oneof though.